core_data_modules/traced_data/traced_data.py

```diff
@@ -42,8 +42,17 @@ def get_call_location():
         frame = inspect.stack()[1]  # Access the previous frame to find out where this function was called from.
         f = frame[1]
         line = frame[2]
-        func = frame[3]
-        return "{}:{}:{}".format(f, str(line), func)
+        name = frame[3]
+        
+        return "{}:{}:{}".format(f, str(line), name)
+
+    @staticmethod
+    def get_function_location(func):
```

Comment:
This function and its test LGTM - could you pull it out into a separate branch and merge it?